src/test/java/TaskListTest.java

```diff
@@ -0,0 +1,36 @@
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+/**
+ * Test class for testing TaskList class.
+ */
+public class TaskListTest {
+    private Storage storage;
+    private Ui ui;
+    private TaskList listOfTasks;
+
+    /**
+     * Tests addTaskTest method of TaskList class.
+     */
+    @Test
+    public void addTaskTest() {
+        listOfTasks = new TaskList();
+        listOfTasks.addTask(new Todo("todo buy bread"));
+        assertEquals("todo buy bread", listOfTasks.getListOfTasks().get(0).getDescription());
+        assertEquals(1, listOfTasks.getSize());
+    }
+
+    /**
+     * Tests removeTask method of TaskList class.
+     */
+    @Test
+    public void removeTaskTest() throws DukeException {
+        listOfTasks = new TaskList();
+        listOfTasks.addTask(new Todo("todo buy bread"));
+        listOfTasks.addTask(new Todo("todo buy butter"));
```

Comment:
Think it's good to assert that the list of tasks here has size 2. (the bug could be that somehow one of the tasks didn't get added in properly)